GCC Administrator (1):
      Daily bump.

Richard Biener (1):
      Bump BASE-VER to 9.4.1

diff --git a/gcc/BASE-VER b/gcc/BASE-VER
index 8148c559489..ccfb75e5120 100644
--- a/gcc/BASE-VER
+++ b/gcc/BASE-VER
@@ -1 +1 @@
-9.4.0
+9.4.1
